sylvia-derive/src/interface.rs

```diff
@@ -17,7 +17,17 @@ use crate::types::msg_variant::MsgVariants;
 mod communication;
 mod mt;
 
-/// Preprocessed `interface` macro input
+/// Preprocessed [`interface`](crate::interface) macro input.
+///
+/// Generates `sv` module containing:
+///     - [Messages](https://cosmwasm-docs.vercel.app/sylvia/macros/generated-types/message-types#interface-messages)
+///         - ExecMsg
+///         - QueryMsg
+///         - SudoMsg
+///     - [MultiTest](https://cosmwasm-docs.vercel.app/sylvia/macros/generated-types/multitest#proxy-trait) helpers
+///     - [Querier](https://cosmwasm-docs.vercel.app/cw-multi-test) trait implementation
+///     - [Executor](https://cosmwasm-docs.vercel.app/cw-multi-test) trait implementation
+///     - Api trait implementation
```

Comment:
Let's add a reference here

Reply:
It's currently missing. Will add that in the future